samples/mixed/provider/src/main.rs

```diff
@@ -135,25 +135,31 @@ async fn register_entities(
     Ok(())
 }
 
+/// Publish.
+///
+/// # Arguments
+/// * `subscription_map` - Subscription map.
+/// * `entity_id` - Entity id.
+/// * `value` - The value to publish.
 async fn publish(subscription_map: Arc<Mutex<SubscriptionMap>>, entity_id: &str, value: &str) {
-    let urls;
+    let uris;
```

Comment:
you can assign this to the result of the block:

```rust
let uris = {
    // get data from lock
    match get_result {
        Some(val) => val.clone(),
        None => HashSet::new(),
    }
};
```

Reply:
Fixed.